### ISSUE TITLE
segment.end is out of boundary of a chunk when we set chunk_length parameter

### ISSUE THREAD

**Issue by ywangwxd** [SYSTRAN/faster-whisper#1204]
Hi, I am making a test of the batched pipeline of transcrible function in faster whisper.  I set chunk_length=5(s), but I found the end time
of the first segment is 5.26.  Shouldn't it be between 0 and 5?

**Comment by MahmoudAshraf97** [comment]
Not necessarily, you are assuming that it started from 0 and this might not be the case, and even in the sequential transcription, silence is removed from the middle so the audio content will be less than the chunk size but the start and the end timestamps can be larger
